conftest.py

```diff
@@ -19,3 +20,17 @@ def pytest_collection_modifyitems(config, items):
     for item in items:
         if "slow" in item.keywords:
             item.add_marker(skip_slow)
+
+
+@pytest.fixture(scope="module")
+def cdnow_data() -> pd.DataFrame:
+    """
+    Load CDNOW_sample.txt into a Pandas dataframe for model and utility function testing.
+    """
+    df = pd.read_csv(
+        "tests/clv/datasets/CDNOW_sample.txt",
```

Comment:
Why not save it as a `csv` directly instead of a `txt`?

Reply:
Consistency with the source data, mainly. I've modified the fixture to reflect the fact this is transaction data, and also provide a link to the original source file:

https://www.brucehardie.com/datasets/

Reply:
Also, is there a reason why `conftest.py` is in the root directory instead of the testing folder?

Reply:
I think it's in the root directory because we run pytest from there? Is that unusual?

Reply:
I think a CLV is much more readable, is there a license problem that doesn't allow us to add the columns?